vmm/src/device_manager.rs

```diff
@@ -210,8 +210,8 @@ pub fn get_win_size() -> (u16, u16) {
 }
 
 pub struct DeviceManager {
-    pub io_bus: devices::Bus,
-    pub mmio_bus: devices::Bus,
+    io_bus: devices::Bus,
+    mmio_bus: devices::Bus,
```

Comment:
Just a note, we need to make those two as `Arc<>` since we're cloning them at the moment, which is not the right thing to do if later we need to add some extra devices on the fly.